app/managers.py

```diff
@@ -0,0 +1,45 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self, db_name: str = "cinema.sqlite") -> None:
+        self.connection = sqlite3.connect(db_name)
+        self.cursor = self.connection.cursor()
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self.cursor.execute("""
+            INSERT INTO actors (first_name, last_name)
+            VALUES (?, ?)
+        """, (first_name, last_name))
+        self.connection.commit()
+
+    def all(self) -> list:
+        self.cursor.execute("SELECT * FROM actors")
+        rows = self.cursor.fetchall()
+        actors = [
+            Actor(
+                id=row[0],
+                first_name=row[1],
+                last_name=row[2]
+            ) for row in rows]
+        return actors
+
+    def update(self, actor_id: int, first_name: str, last_name: str) -> None:
+        if first_name:
```

Comment:
If you adding a check here for first_name and last_name, then add to annotation that they are optional
And what if user will first_name and last_name?

Reply:
Did you change my comment?